.github/workflows/stackhpc-all-in-one.yml

```diff
@@ -243,6 +268,13 @@ jobs:
         run: |
           test $(wc -l < tempest-artifacts/failed-tests) -lt 1
 
+      - name: Upload logs artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: kolla-logs-${{ inputs.os_distribution }}-${{ inputs.os_release }}-${{ inputs.neutron_plugin }}
+          path: logs.zip
```

Comment:
I don't see where this is created.

Reply:
My mistake, I just left that in from when I was debugging things in the past. I don't think it ever worked to be honest.